runtime/tsan.c

```diff
@@ -395,7 +390,15 @@ CAMLreally_no_tsan void __tsan_unaligned_volatile_read##size(void *ptr)        \
 }                                                                              \
 CAMLreally_no_tsan void __tsan_volatile_write##size(void *ptr)                 \
 {                                                                              \
-  __tsan_write##size(ptr);                                                     \
+  const bool is_atomic = size <= sizeof(long long) &&                          \
+             is_aligned(ptr, 8);                                               \
```

Comment:
Nitpick: I find the indentation choice here confusing. I would expect you to either 2-align, or align on `size`, but the visual alignment of `is_` looks intentional and carries no meaning that I can see.

Reply:
Indeed, it didn’t make sense. Fixed in 4fd66b1.